setup.py

```diff
@@ -241,6 +241,7 @@ def run(self):
     'spyder-kernels>=2.2.0,<2.3.0',
     'textdistance>=4.2.0',
     'three-merge>=0.1.1',
+    'traitlets>=4.1',
```

Comment:
```suggestion
```

`trailets` is a transitive dependency, so we don't need to depend on it directly:

    spyder -> spyder-kernels -> ipykernel -> traitlets

Reply:
This is true, though the remainder of the patch does explicitly import it.  I guess it's a matter of style to a great extent; this explicit listing just makes the version requirements explicit, though ipykernel does that anyway.  I'm not particularly bothered either way; happy to make that change or let you do it.

Reply:
Please make it. We already have a lot of dependencies to add one more for such a simple change.

Reply:
Done!